edi_voxel_sale_secondary_unit_oca/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Edi Voxel Sale Secondary Unit Oca",
+    "summary": "Map Voxel UoM to Sale Secondary UoM and Use Them",
+    "version": "15.0.1.0.0",
+    "development_status": "Production/Stable",
```

Comment:
```suggestion
    "development_status": "Beta",
```
The development status of the dependencies is `Beta`